### PR TITLE
Updating awx_test.yml to respect both web and task deployments.

### PR DESCRIPTION
**Pull request by djyasin**
##### SUMMARY

Updated these tests: molecule/default/tasks/awx_test.yml to validate both web and task containers. With the split of web and task the name block needs to acknowledge each pod respectively.

Fixes issue #1315 
##### ISSUE TYPE

 - Bug, Docs Fix or other nominal change

### CODE REVIEW

File: molecule/default/tasks/awx_test.yml
Comment (TheRealHaoLiu) [inline molecule/default/tasks/awx_test.yml:1]: this is the AWX resource not the web deployment

File: molecule/default/templates/awx-deploy.yml
Comment (TheRealHaoLiu) [inline molecule/default/templates/awx-deploy.yml:1]: why is this file added?

**Comment by djyasin** [comment]
I will definitely squash my commits once the resolution has been finalized!

File: molecule/default/tasks/awx_test.yml [hunk 1]
```diff
@@ -109,14 +118,22 @@
...
           }}
 
+    - name: Extract additional_labels from AWX Task Pod
+      set_fact:
+        pod_additional_labels: >-
+          {{ awx_task_pod.resources[0].metadata.labels
+              | dict2items | selectattr('key', 'in', this_awx.resources[0].spec.additional_labels)
+              | list
+          }}
+
```
Comment (TheRealHaoLiu) [inline molecule/default/tasks/awx_test.yml:142]: after we extracted the label we need to validate the label see the next task

**Review by thedoubl3j — APPROVED** [review]

**Review by TheRealHaoLiu — APPROVED** [review]
